packages/react-widgets/test/DateTimePicker-test.js

```diff
@@ -201,13 +202,14 @@ describe('DateTimePicker', () => {
     wrapper.find('.rw-i-calendar').simulate('click')
 
     setTimeout(() => {
-      expect(wrapper.instance()._values.open).to.not.equal(true)
+      expect(instanceRef.current.props.open).to.not.equal(true)
```

Comment:
I don't think these are testing the right thing anymore, `props` won't reflect the controlled values because they are stored in store in the Component. We need to test these conditions more generally since state is no longer accessible even with a ref. 

My suggestion would be to control the tested values and assert that the callbacks are not called, e.g. `onToggle` here.

Reply:
Good call. I'll go back through the test cases and fix things up.

Reply:
I've fixed up the test cases in DateTimePicker-test.js

They're ready for you to review. Thanks!